### DISCUSSION TITLE
Plans to support svelte+pocketbase (typescript and javascript)

### DISCUSSION THREAD

**Discussion by miguelgargallo** [keycloak/keycloak#20499] [Ideas]
Are there anu plans to support svelte+pocketbase (typescript and javascript)?

**Comment by jonkoops** [comment]
No, there are not. We already have JavaScript libraries you can integrate yourself in your framework of choice, or you can use any OpenID Connect compatible client library.

**Comment by miguelgargallo** [comment]
The idea is that I have my own port Svelte+Pocketbase to KeyCloack, the idea is to know if something official is undergoround... thanks for your time!
